.github/workflows/demo-server.yml

```diff
@@ -42,51 +48,24 @@ jobs:
-          GROUP: ${{ secrets.AZURE_DEMO_RESOURCEGROUP }}
-        run: |
-          az webapp config container set --resource-group $GROUP --name $APP --docker-registry-server-user ${{ secrets.DOCKER_REGISTRY_USERNAME }} --docker-registry-server-password ${{ secrets.DOCKER_REGISTRY_PASSWORD }} --docker-custom-image-name $DOCKER_IMAGE_UNIVERSAL || az webapp create --resource-group $GROUP --plan ${{ secrets.AZURE_DEMO_APPSERVICEPLAN }} --name $APP --docker-registry-server-user ${{ secrets.DOCKER_REGISTRY_USERNAME }} --docker-registry-server-password ${{ secrets.DOCKER_REGISTRY_PASSWORD }} --deployment-container-image-name $DOCKER_IMAGE_UNIVERSAL
-          az webapp config appsettings set -g $GROUP -n $APP --settings LOGGING=true ICM_BASE_URL=$ICM_BASE_URL THEME=default
-          az webapp deployment container config -g $GROUP -n $APP --enable-cd true
-          echo "B2C channel: http://$APP.azurewebsites.net"
-
-      - name: Create or Update B2B App Service
+      - name: Create or Update containerized WebApp
         env:
-          APP: '${{ secrets.AZURE_DEMO_RESOURCEGROUP }}-${{ github.event.pull_request.number }}-${{ env.B2B_APP_SUFFIX }}'
+          APP: ${{ env.AZURE_WEBAPP_NAME }}
           GROUP: ${{ secrets.AZURE_DEMO_RESOURCEGROUP }}
         run: |
           az webapp config container set --resource-group $GROUP --name $APP --docker-registry-server-user ${{ secrets.DOCKER_REGISTRY_USERNAME }} --docker-registry-server-password ${{ secrets.DOCKER_REGISTRY_PASSWORD }} --docker-custom-image-name $DOCKER_IMAGE_UNIVERSAL || az webapp create --resource-group $GROUP --plan ${{ secrets.AZURE_DEMO_APPSERVICEPLAN }} --name $APP --docker-registry-server-user ${{ secrets.DOCKER_REGISTRY_USERNAME }} --docker-registry-server-password ${{ secrets.DOCKER_REGISTRY_PASSWORD }} --deployment-container-image-name $DOCKER_IMAGE_UNIVERSAL
-          az webapp config appsettings set -g $GROUP -n $APP --settings LOGGING=true ICM_BASE_URL=$ICM_BASE_URL THEME=blue TACTON='${{ secrets.TACTON }}'
+          az webapp config appsettings set -g $GROUP -n $APP --settings LOGGING=true ICM_BASE_URL=$ICM_BASE_URL
           az webapp deployment container config -g $GROUP -n $APP --enable-cd true
-          echo "B2B channel: http://$APP.azurewebsites.net"
 
       - name: Create Commit Comment
-        env:
-          APP: '${{ secrets.AZURE_DEMO_RESOURCEGROUP }}-${{ github.event.pull_request.number }}'
         uses: peter-evans/commit-comment@v1
         with:
           body: |
             Azure Demo Servers are available:
-            - [Universal B2C](http://${{ env.APP }}-${{ env.B2C_APP_SUFFIX }}.azurewebsites.net)
-            - [Universal B2B](http://${{ env.APP }}-${{ env.B2B_APP_SUFFIX }}.azurewebsites.net)
+            - [Universal B2C](http://${{ env.AZURE_WEBAPP_NAME }}.azurewebsites.net/home)
+            - [Universal B2B](http://${{ env.AZURE_WEBAPP_NAME }}.azurewebsites.net/home;theme=blue)
```

Comment:
I totally forgot that this works now 😅

Reply:
Means potentially twice as much demo deployments under one service plan

Reply:
The service plan is the bill not how many services are deployed within 😉